lib/inspec/reporters.rb

```diff
@@ -9,26 +9,26 @@
 
 module Inspec::Reporters
   # rubocop:disable Metrics/CyclomaticComplexity
-  def self.render(reporter, run_data)
+  def self.render(reporter, run_data, opts = {})
```

Comment:
I think what's interesting here, is wouldn't this setup give us the power to use options on any of our reporters in the future? (With the first two being these specific ones relating to JSON)

Though I do see where we could be increasing data coupling (passing data to the reporters) -  it is with a single options hash that opens up these future feature possibilities.

Reply:
At this point, the `reporter` passed to this method is `["json", {"stdout"=>true}]`
That might be an existing hash that we can and should use to pass options for the reporter